docker/optimizer/localOptimize.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
```

Comment:
Can we copy from the dist module in docker/optimizer/build.sh so that we don't have to maintain an extra copy here?

Reply:
It seems that this file is exactly the same as `dist/src/main/arctic-bin/bin/localOptimize.sh`. It is suggested to copy this file in build.sh by using the CP command.